packages/components/react-web/src/Layout/Frame/Frame.jsx

```diff
@@ -1,29 +1,37 @@
 import PropTypes from 'prop-types';
 
-
-import { cx, css } from '@emotion/css';
-
 import frameStyles from './Frame.styles';
 
-export const Frame = ({ ratio, objectPosition, ...props }) => {
-  const matches = ratio && ratio !== null ? ratio.match(/(\d+):(\d+)/) : null;
-
-  const [denominator, numerator] =
-    matches && matches.length > 0 ? matches.slice(1) : [undefined, undefined];
+export const Frame = ({
+  ratio,
+  objectPosition,
+  children,
+  className,
+  ...props
+}) => {
+  const [, denominator, numerator] = ratio.match(/(\d+):(\d+)/) ?? [];
 
   return (
     <div
       css={frameStyles({ denominator, numerator, objectPosition })}
-      className={`gds-frame ${props.className ? props.className : ''}`}
+      className={`gds-frame ${className}`}
```

Comment:
This will now become `gds-frame null` when no class is provided and the default is used
It should be just `gds-frame`